### PR TITLE
fix(event-display): allowing geometry name to be undefined

### PR DESCRIPTION
**Pull request by kjvbrt**

### CODE REVIEW

File: packages/phoenix-event-display/src/event-display.ts [hunk 1]
```diff
@@ -359,7 +359,7 @@ export class EventDisplay {
    */
   public async loadGLTFGeometry(
     url: any,
-    name: string,
```
Comment (9inpachi) [inline packages/phoenix-event-display/src/event-display.ts:362]: Have you tested how the behavior is without the name? The name appears in the Phoenix menu and can potentially make it hard to navigate through the menu if there is no name.

Comment (EdwardMoyse) [reply packages/phoenix-event-display/src/event-display.ts:362]: Test

**Comment by kjvbrt** [comment]
Yes, but it is equally hard if you have only one name:
image

I'm not sure what is the preferred behaviour. To have the `name` variable as backup if the file does not contain the scene name or to overwrite the scene name from the file.

File: packages/phoenix-event-display/src/event-display.ts (hunk 1 above)
Comment (EdwardMoyse) [reply packages/phoenix-event-display/src/event-display.ts:362]: @kjvbrt is trying to reply... but in short, if he does not make it undefined then the passed name overrides the names in the geometry

Comment (EdwardMoyse) [reply packages/phoenix-event-display/src/event-display.ts:362]: I think we have to accept this.

**Pull request merged**